src/lwip/patches/README.md

```diff
@@ -21,11 +21,13 @@ recent API change on upstream LwIP. The previous version of this function is
 
 ### ND6 LLADDR fix
 
-This patch fixes a bug where the RA processing fails if the RA includes an
-LLADDR option with a hw address that is not the max length. This happens for
-thread devices.
+In earlier version of lwIP, there is a bug where the RA processing fails if the
+RA includes an LLADDR option with a hw address that is not the max length. This
+happens for thread devices. Relevant patches from lwIP main that should be
+applied back to any Matter device using an older version of lwIP:
 
--   patch file: nd6_lladdr_fix.patch
+https://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=7807f706f357d2eea85d9927d970991dc4b1ef90
+https://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=7807f706f357d2eea85d9927d970991dc4b1ef90
```

Comment:
Both of these lines are the same commit. Is that intended?

Reply:
nope, fixed.